server/models/comment.model.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const Comment = sequelize.define('Comment', {
+    user_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    },
+    gift_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    },
+    body: {
+      type: DataTypes.TEXT,
+      allowNull: true
+    },
+    parent_id: {
+      type: DataTypes.INTEGER,
+      allowNull: true
+    },
+    createdAt: {
+      type: DataTypes.BIGINT,
+      allowNull: true
+    },
+    updatedAt: {
+      type: DataTypes.BIGINT,
+      allowNull: true
+    }
+  }, {
+    paranoid: false,
```

Comment:
Delete paranoid property, because it unneeded.
I'll correct it in another models.